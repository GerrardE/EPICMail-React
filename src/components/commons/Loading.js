import React from 'react';
import loader from './material-loader.gif';

export const Loading = () => {
  return (
    <div>
      <img src={loader} alt="loading..." className="loader"></img>
    </div>
  )
}
