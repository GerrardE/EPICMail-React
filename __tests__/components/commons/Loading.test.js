import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loading from '../../../src/components/commons/Loading';

const props = {
  loading: ''
}

const setup = () => {
  const wrapper = shallow(<Loading {...props} />);
  return wrapper
}

describe('<Loading />', () => {
  test('should render the Loading component properly', () => {
    let wrapper;
    wrapper = setup();
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})
