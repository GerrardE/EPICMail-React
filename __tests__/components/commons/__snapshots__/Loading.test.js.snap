// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render the Loading component properly 1`] = `
<div
  className="middle"
>
  <h5
    className="lead-title"
  >
    Loading ...
  </h5>
  <img
    alt="loading..."
    className="loader"
    src={
      Object {
        "process": [Function],
      }
    }
  />
</div>
`;
